### PR TITLE
Ads updates

### PR DESCRIPTION
**Pull request by dwick**
Related changes in the ads plugin:
https://github.com/reddit/reddit-plugin-adzerk/commit/f435cf06524cfa73763b069f9dc6e18a4fd490f6
https://github.com/reddit/reddit-plugin-adzerk/commit/5e7bc4ce6ca8136f1b9b021f3b0c2295abd600da

:eyeglasses: @ajacksified Let me know if you want someone else on the team to look at this.

### CODE REVIEW

File: src/views/components/ListingList.jsx [hunk 1]
```diff
@@ -116,9 +120,16 @@ class ListingList extends BaseComponent {
   }
 
   buildAd() {
-    const srnames = uniq(this.props.listings.map(function(l) {
-      return l.subreddit;
-    }));
+    let site;
```
Comment (ajacksified) [inline src/views/components/ListingList.jsx]: default to `site = FRONTPAGE_NAME` and drop the `else {` below

**Comment by ajacksified** [comment]
Minor change, looks good otherwise. I'll look into why travis is breaking. Probably a bad minor version of a build dependency =/

:nail_care:

**Comment by dwick** [comment]
:haircut: 

ooo, you guys have CI!

File: src/views/components/ListingList.jsx [hunk 2]
```diff
@@ -12,6 +12,10 @@ import Listing from '../components/Listing';
 const Proptypes = React.PropTypes;
 
 const _AD_LOCATION = 11;
+const FRONTPAGE_NAME = ' reddit.com';
```
Comment (uzi) [inline src/views/components/ListingList.jsx]: Is the extra space here intentional?

Comment (dwick) [inline src/views/components/ListingList.jsx]: yes

**Comment by ajacksified** [comment]
:fish:

**Pull request merged**